eZ/Bundle/EzPublishCoreBundle/Command/ReindexCommand.php

```diff
@@ -69,11 +76,22 @@ class ReindexCommand extends ContainerAwareCommand
     public function initialize(InputInterface $input, OutputInterface $output)
     {
         parent::initialize($input, $output);
-        $this->searchIndexer = $this->getContainer()->get('ezpublish.spi.search.indexer');
+
+        $searchIndexer = $this->getContainer()->get('ezpublish.spi.search.indexer');
```

Comment:
This is obsolete, unless you have solution ready to apply when merging into master, I won't accept it right now. It needs to be injected.